docs/source/configuration.rst

```diff
@@ -197,6 +199,37 @@ Examples
             message: |
               The pull request base branch has reached end-of-life.
 
+
+.. _relative timestamp:
+
+Relative Timestamp
+~~~~~~~~~~~~~~~~~~
+
+Timestamps can be expressed relative to the current date and time.
+The format is ``[DD days] [HH:MM] ago``:
+
+* DD, the number of days
+* HH, the number of hours
+* MM, the number of minutes
+
+If the current date is 18th June 2020, ``updated-at>=14 days ago`` will be translated ``updated-at>=2020-06-04T00:00:00``.
+
+Examples
+++++++++
+
+.. code-block:: yaml
+
+      - name: close stale pull request
+        conditions:
+          - base=main
+          - -closed
+          - updated-at>=14 days ago
```

Comment:
```suggestion
          - updated-at<14 days ago
```

Funny that you actually made the mistake 😆

Reply:
I copy/paste the other PR that have the wrong implementation.